src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -276,52 +272,75 @@
+                async with load_workflow(workflow_config_file_path) as local_session_manager:
+                    output: EvaluationRunOutput = await eval_runner.run_and_evaluate(
+                        session_manager=local_session_manager, job_id=job_id)
+
+                if output.workflow_interrupted:
+                    await job_store.update_status(job_id, "interrupted")
+                else:
+                    parent_dir = os.path.dirname(output.workflow_output_file) if output.workflow_output_file else None
+
+                    await job_store.update_status(job_id, "success", output_path=str(parent_dir))
+            except Exception as e:
+                logger.exception("Error in evaluation job %s: %s", job_id, str(e))
```

Ruff (0.12.2)

Redundant exception object included in `logging.exception` call

(TRY401)